### ISSUE TITLE
Cannot place some furniture with a block on the right face

### ISSUE THREAD

**Issue by arrisar** [sketchmacaw/MacawsModsIssues#78]
**Description**
When attempting to place the large bookcase (and some other items), any blocks on the right side are considered blocking.

Consider the 2x2 bookcase as B (with the placement location as P), air as A, and other non-air walls as W in the below examples:

```
# blocking, cannot place
WBBW
WBPW

# also blocking
ABBW
ABPW

# non-blocking, nothing around it
ABBA
ABPA

# non-blocking, but the exact same as ex 2
WBBA
WBPA
```

This also affects roof placement with rear blocks, as mention in #32.

**Workaround**
By removing the walls on the side, you can place the furniture, then simply replace the furniture.

**Suggestion**
Do not consider blocks on the right face of the furniture being placed to be blocking.

**Labels**
1.16.5, Macaw's Furniture, Bug

**Comment by lincore81** [comment]
I have observed a similar behaviour with furniture like the cupboard dresser, which will not tolerate blocks to its left. Coincidentally, the origin is the left block, whereas for the 2x2 furniture it's the lower right block. This inconsistency might be a problem in itself.

(mcwfurniture 2.0.1)

**Comment by sketchmacaw** [comment]
Fixed in 3.0.0 Update
